staging/src/k8s.io/component-base/metrics/registry.go

```diff
@@ -49,6 +49,50 @@ func shouldHide(currentVersion *semver.Version, deprecatedVersion *semver.Versio
 	return false
 }
 
+func validateShowHiddenMetricsVersion(currentVersion semver.Version, targetVersionStr string) error {
+	if targetVersionStr == "" {
+		return nil
+	}
+
+	// We assume the input target version with the format `x.y`, here simply suffix `0` as patch version.
+	// It helps to compare with current version.
+	pathVersion := fmt.Sprintf("%s.0", targetVersionStr)
+
+	targetVersion, err := semver.Parse(pathVersion)
+	if err != nil {
+		return fmt.Errorf("specified --show-hidden-metrics %v not follows the version format x.y", targetVersionStr)
```

Comment:
Oh, my god.  I forgot to update this flag name in error message.
Same as below.